doc/source/ray-core/tasks/using-ray-with-gpus.rst

```diff
@@ -95,17 +95,19 @@ Workers not Releasing GPU Resources
 **Note:** Currently, when a worker executes a task that uses a GPU (e.g.,
 through TensorFlow), the task may allocate memory on the GPU and may not release
 it when the task finishes executing. This can lead to problems the next time a
-task tries to use the same GPU. You can address this by setting ``max_calls=1``
-in the remote decorator so that the worker automatically exits after executing
-the task (thereby releasing the GPU resources).
+task tries to use the same GPU. To address the problem, Ray disables the worker
+process reuse between tasks by default, where the GPU resources is released after
+the task process exists. Since this adds overhead to GPU tasks scheduling,
+you can re-enable worker reuses by setting ``max_calls=0``
```

Comment:
```suggestion
the task process exists. Since this adds overhead to GPU task scheduling,
you can re-enable worker reuse by setting ``max_calls=0``
```